test/vendor/tracekit.test.js

```diff
@@ -134,6 +134,39 @@ describe('TraceKit', function(){
             }
         });
 
+        describe('when no 5th argument (error object)', function () {
+            it('should seperate name, message for default error types (e.g. ReferenceError)', function (done) {
+                subscriptionHandler = function (stackInfo, extra) {
+                    assert.equal(stackInfo.name, 'ReferenceError');
+                    assert.equal(stackInfo.message, 'foo is undefined');
+                    done();
+                };
+                TraceKit.report.subscribe(subscriptionHandler);
+                window.onerror('Uncaught ReferenceError: foo is undefined', 'http://example.com', 1337);
```

Comment:
It is browser specific. Chrome/Android browser add `Uncaught`. Firefox does not.

I'm just emulating what we do if there is an `Error` object (which has a `name` property). The `name` property does not include "uncaught".